pandas/tests/indexes/interval/test_indexing.py

```diff
@@ -158,6 +158,20 @@ def test_get_loc_decreasing(self, values):
         expected = 0
         assert result == expected
 
+    def test_get_loc_non_scalar_errors(self):
```

Comment:
Can you parametrize this test over `key` so we only need the `pytest.raises` logic once?  Also, can you move the comment with the github issue number to be the first thing in the test, e.g.
```python
@pytest.mark.parametrize("key", [[5], (2, 3)])
def test_get_value_errors(self, key):
    # GH 31117
    idx = ...
```

Reply:
changed